community/xbacklight/sources

```diff
@@ -0,0 +1 @@
+https://gitlab.freedesktop.org/xorg/app/xbacklight/-/archive/xbacklight-1.2.3/xbacklight-xbacklight-1.2.3.tar.gz
```

Comment:
You should always use a release tarball. 

```
https://www.x.org/releases/individual/app/xbacklight-1.2.3.tar.bz2
```

This source does not require `autogen` or `libtool`.

Reply:
the source and build have been changed